release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -71,17 +71,33 @@ For more information, see xref:../installing/cluster-capabilities.adoc#cluster-c
 [id="ocp-4-16-web-console"]
 === Web console
 
+[id="ocp-4-16-patternfly4-deprecated"]
+==== Patternfly 4 is now deprecated with 4.16
+With this release, Patternfly 4 and React Router 5 is depreciated in the web console. All plugins should migrate to Patternfly 5 and React Router 6 as soon as possible.
+
 [id="ocp-4-16-administrator-perspective"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
-//Are there any for 4.16
+
+* A Google Cloud Platform (GCP) token authorization was added to the *Infrastructure features* filter in the OperatorHub.
+* A new quick start is available with information on the impersonate workflow and functionality for users with impersonare access.
 
 [id="ocp-4-16-developer-perspective"]
 ==== Developer Perspective
 
 This release introduces the following updates to the *Developer* perspective of the web console:
-//* are there any for 4.16?
+
+* When searching,
+A new section was added to the *Search* menu to display the recently searched items in the order they were searched.
```

Comment:
Can we be more specific about the search menu?
```suggestion
A new section was added to the *Resources* dropdown on the search page to display the recently searched items in the order they were searched.
```

Reply:
Yes I like that! Thanks